charlotte_core/src/main.rs

```diff
@@ -3,16 +3,45 @@
 
 mod access_control;
 mod arch;
+mod framebuffer;
 mod bootinfo;
 
+use framebuffer::*;
```

Comment:
Should avoid a wildcard import, we only need to pull in `framebuffer::framebuffer{Points, init_framebuffer}`